### DISCUSSION TITLE
Skyline

### DISCUSSION THREAD

**Discussion by TheAlex0110** [skyline-emu/skyline#2047] [General]
Que sea compatible con android 9.0 en adelante

**Comment by cristhiantorr** [comment]
Hola buenas noches no entendí que lo que dijiste

**Reply by JERD001** [reply]
No va a pasar, lo más probable es que si te quedaste en esa versión tu celular no sea lo suficientemente potente para Skyline. Suponiendo que tu fabricante simplemente no quiso actualizar aún así Android 9 le faltan funciones de las que skyline depende, nunca será compatible con esa versión.

**Comment by TheAlex0110** [comment]
Mi cell es un S8 4RAM Snap Dragon(835) es una Lastima que no lo pueda difrutar Enviado desde mi Galaxy
-------- Mensaje original --------De: Jonatan ***@***.***> Fecha: 5/12/22  7:34 a.m.  (GMT-05:00) A: skyline-emu/skyline ***@***.***> CC: TheAlex0110 ***@***.***>, Author ***@***.***> Asunto: Re: [skyline-emu/skyline] Skyline (Discussion #2047) 

No va a pasar, lo más probable es que si te quedaste en esa versión tu celular no sea lo suficientemente potente para Skyline. Suponiendo que tu fabricante simplemente no quiso actualizar aún así Android 9 le faltan funciones de las que skyline depende, nunca será compatible con esa versión.
